packages/react-components/react-message-bar-preview/src/components/MessageBarActions/renderMessageBarActions.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsxRuntime automatic */
+/** @jsxImportSource @fluentui/react-jsx-runtime */
+
+import { assertSlots } from '@fluentui/react-utilities';
+import type { MessageBarActionsState, MessageBarActionsSlots } from './MessageBarActions.types';
+import { ButtonContextProvider } from '@fluentui/react-button';
+
+/**
+ * Render the final JSX of MessageBarActions
+ */
+export const renderMessageBarActions_unstable = (state: MessageBarActionsState) => {
+  assertSlots<MessageBarActionsSlots>(state);
+  if (state.layout === 'multiline') {
+    return (
+      <ButtonContextProvider value={{ size: 'small' }}>
```

Comment:
- This should be a memoized value as `ButtonContextProvider` is not using context selectors
- The value should be defined in `useMessageBarContextValues`